app/controllers/api/requirement_templates_controller.rb

```diff
@@ -36,14 +36,32 @@ def create
                      "requirement_template.create_success",
                      { blueprint: RequirementTemplateBlueprint }
     else
-      render_error Constants::Error::REQUIREMENT_TEMPLATE_CREATE_ERROR,
+      render_error "REMOVE ME AFTER REBASE",
```

Comment:
Double check this - has rebase happened? should the restore error capture also be removed?